vllm/model_executor/models/nemotron_h.py

```diff
@@ -552,10 +554,18 @@ def forward(self,
                 mamba_state_shape = \
                     self.get_mamba_state_shape_from_config(
                         self.vllm_config, use_v1=False)
-                self.mamba_cache = MambaCacheManager(self.vllm_config,
-                                                     self.lm_head.weight.dtype,
-                                                     num_mamba_layers,
-                                                     *mamba_state_shape)
+
+                mamba_ssm_cache_dtype = get_kv_cache_torch_dtype(
+                    self.vllm_config.cache_config.mamba_ssm_cache_dtype,
+                    self.lm_head.weight.dtype)
```

Comment:
`MambaCacheManager` is used by V0 code only. Is the intention that this PR will be V0-only or do you also intend to support fp32 mamba state in V1?